resources/functionMap.php

```diff
@@ -3334,7 +3334,7 @@
 'getimagesize' => ['array{0:int, 1: int, 2: int, 3: string, mime: string, channels?: int, bits?: int}|false', 'imagefile'=>'string', '&w_info='=>'array'],
 'getimagesizefromstring' => ['array{0:int, 1: int, 2: int, 3: string, mime: string, channels?: int, bits?: int}|false', 'data'=>'string', '&w_info='=>'array'],
 'getlastmod' => ['int|false'],
-'getmxrr' => ['bool', 'hostname'=>'string', '&w_mxhosts'=>'array', '&w_weight='=>'array'],
+'getmxrr' => ['bool', 'hostname'=>'string', '&w_mxhosts'=>'list<string>', '&w_weight='=>'list<0, 65535>'],
```

Comment:
What effect should this have? The types aren't enforced: https://phpstan.org/r/25c3b187-4888-4aa4-86ad-5e350f16346f